### PR TITLE
#1102 Ch sequelize cli postgresql 1102

### PR DESCRIPTION
**Pull request by kwizeradev**
### TEST#1 sequelize-cli-postgres-1102 PREQ
1.
i. Make sure that the src/database/models is empty
   ii. Make sure that the src/database/seeders is empty

 ### TEST#2 sequelize-cli-postgres-1102 COMMANDS

- TO CREATE USER MODAL : npm run create-model
- TO MIGRATE : npm run migrate
- TO UNDO ALL MIGRATIONS : npm run undo-migrate
- TO CREATE SEEDER : npm run create-seed
- TO FEED SEEDER use ./data.txt
- TO EXECUTE SEEDER : npm run exec-seed
- TO UNDO CREATED SEED : npm run undo-seed

More information : https://sequelize.org/docs/v6/other-topics/migrations/

### CODE REVIEW

**Review by tjeaneric — APPROVED** [review]

**Review by Kalimbak — APPROVED** [review]

**Review by Angelus123 — APPROVED** [review]

File: data.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          updatedAt: new Date(),
+        },
+      ],
+      {}
+    );
+
+
+
+
+    await queryInterface.bulkDelete('users', null, {});
```
Comment (furebo) [inline data.txt]: Please delete this file since you have seeders in seeders folders

File: package.json [hunk 2]
```diff
@@ -7,6 +7,12 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "better-npm-run dev",
     "start": "npm run build && better-npm-run prod",
+    "create-model": "npx sequelize-cli model:generate --name user --attributes name:string,username:string,email:string,password:string",
```
Comment (furebo) [inline package.json]: Remove the create_model command because it will create the user model only once.

File: data.txt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        },
+        {
+          id: 2,
+          name: 'user2',
+          username: 'username2',
+          email: 'user2@test.com',
+          password: Math.floor(Math.random() * 100),
+          createdAt: new Date(),
+          updatedAt: new Date(),
+        },
```
Comment (furebo) [inline data.txt]: Please remove this file data.txt because you have the seeders.

**Review by furebo — CHANGES_REQUESTED** [review]
Rebase the develop and put database credentials in environment variables since now the dotenv is now configured.

**Review by furebo — CHANGES_REQUESTED** [review]
Please also remember to squash the commits

File: .sequelizerc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require("@babel/register");
+
+const path = require('path');
+
+module.exports = {
+    "config": path.resolve('./src/config', 'config.json'),
```
Comment (furebo) [inline .sequelizerc]: Here we need to use config.js because we have it there,instead of json file

**Review by furebo — CHANGES_REQUESTED** [review]

**Review by furebo — APPROVED** [review]

**Pull request merged**